.github/workflows/contract_requiring_verification_published.yml

```
@@ -14,7 +14,7 @@ on:
         required: true
 
 env:
-  PACT_BROKER_BASE_URL: https://test.pactflow.io
+  PACT_BROKER_BASE_URL: ${{ secrets.PACT_BROKER_BASE_URL }}
```

```suggestion
  PACT_BROKER_BASE_URL: ${{ vars.PACT_BROKER_BASE_URL }}
```

The base URL is unlikely to be sensitive information. This also ensures that the secret is _not_ scrubbed from logs, which can help with debugging.

---

Thanks, I've added the variable now also so when this is updated it should work correctly. 

Could you please make https://github.com/pactflow/example-provider/blob/master/.github/workflows/build.yml#L10 a var also?